src/style.css

```
@@ -1 +1,46 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url(/src/images/star-active.svg);
+}
+
```

```suggestion
.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2) {
  background-image: url(/src/images/star-active.svg);
}
```
combine it using the comma